docs/support_table.md

```diff
@@ -2,7 +2,7 @@
 id: support_table
 title: Support Table
 ---
-<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.10.0-beta/dist/katex.min.css" integrity="sha384-9tPv11A+glH/on/wEu99NVwDPwkMQESOocs/ZGXPoIiLE8MU/qkqUcZ3zzL+6DuH" crossorigin="anonymous">
+{@stylesheet: katex.min.css}
```

Comment:
Where is this feature documented? It seems a bit weird, e.g., the output includes `<meta property="og:description" content="{@stylesheet: katex.min.css}">`.  (Maybe we can manually include a description in the YAML header?)

Is there a reason to do it this way vs. via a `<link>` tag?  Avoiding the absolute path issue?  Or does it embed the CSS in another way?  (I haven't actually figured out how the CSS gets in there...)

Reply:
@edemaine This is `remarkable-embed` plugin and is to fix absolute path issue (#1525 and #1520). The `<link>` tag is in `body > div.navPusher > div > div.container.mainContainer > div > div.post > article > div > span > p > link`.

Reply:
Got it; all makes sense now.  Can we fix the description thing?

Reply:
@edemaine I think we can move stylesheet after the first paragraph.

Reply:
That seems to fix the issue.